helm/chart/values.yaml

```diff
@@ -79,6 +79,7 @@ lifecycleOperator:
       optionsControllerLogLevel: "0"
       otelCollectorUrl: otel-collector:4317
       pythonRunnerImage: ghcr.io/keptn/python-runtime:v1.0.0
+      schedulingGatesEnabled: false
```

Comment:
this is a config option that involves two components: lifecycle-operator and scheduler, I would not leave it under this single env var but have it higher in a global config part of the chart